AGENTS.md

```
@@ -96,6 +96,8 @@ Read each folder's README file for more details
 - Use `safeAsync` when using `await` to better handle errors
 - When creating a skeleton, use the `Skeleton` component from `@zoonk/ui/components/skeleton`
 - Always build skeleton components when using `Suspense` for loading states
+- Never import model types from `@zoonk/db` directly. Always import them from `@zoonk/core` instead. Only `@zoonk/core/types` should import and consume from `@zoonk/db`
+- When creating components, follow the best practices from `components.build`, especially following a composable architecture: https://www.components.build/llms.txt
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)